extension/src/setup/index.ts

```diff
@@ -389,18 +390,14 @@ export class Setup
     }
   }
 
-  private isDVCBeingUsedGlobally() {
```

Comment:
We append the path to python to the `PATH` variable for the edge case that DVC is being used as a binary but with a virtual environment.

Reply:
because we append the virtual environment path to the PATH environment variable it is technically being used.